### PR TITLE
[GUVNOR-2279] Exposing MavenSession.

### PR DESCRIPTION
**Pull request by afcollins**
- Exposing MavenSession from MavenEmbedder.
- Refactored MavenProjectLoader.getMavenEmbedder() to expose
  MavenEmbedder.

### CODE REVIEW

File: kie-ci/src/main/java/org/kie/scanner/embedder/MavenProjectLoader.java [hunk 1]
```diff
@@ -50,14 +50,26 @@ public static MavenProject parseMavenPom(InputStream pomStream) {
...
         }
     }
 
+    public static MavenEmbedder newMavenEmbedder(boolean offline) {
+        MavenRequest mavenRequest = createMavenRequest(offline);
+        MavenEmbedder mavenEmbedder;
+            try {
+                mavenEmbedder = new MavenEmbedder( mavenRequest );
+            } catch (MavenEmbedderException e) {
+                e.printStackTrace();
```
Comment (mrietveld) [inline kie-ci/src/main/java/org/kie/scanner/embedder/MavenProjectLoader.java]: If you could add a (SLF4j) logger statement here, with an appropriate message, that would be much appreciated. 

Something like: 

`logger.error( "Unable to create new MavenEmbedder instance: " + e.getMessage, e );`

The last argument makes sure that the stack trace is printed. 

Thanks!

Comment (mrietveld) [inline kie-ci/src/main/java/org/kie/scanner/embedder/MavenProjectLoader.java]: This also applies to the related guvnor PR.

**Comment by manstis** [comment]
@mariofusco Hi Mario, could you please review this PR?

**Comment by mariofusco** [comment]
Merged by https://github.com/droolsjbpm/drools/commit/40e17580a

**Pull request closed without merging**